Microservices.PlatformService/Profiles/PlatformProfiles.cs

```diff
@@ -0,0 +1,26 @@
+﻿using AutoMapper;
+using Microservices.PlatformService.Dtos;
+using Microservices.PlatformService.Models;
+
+namespace Microservices.PlatformService.Profiles
+{
+    public class PlatformProfiles : Profile
+    {
+        public PlatformProfiles()
+        {
+            CreateMap<PlatformReadDto, Platform>()
+                .ReverseMap();
+
+            CreateMap<PlatformCreateDto, Platform>()
```

Comment:
See comment above.

Reply:
It's used at Presentation Layer and contains validation attributes.